browser/ui/webui/brave_shields/shields_panel_data_handler.cc

```diff
@@ -184,18 +175,22 @@ void ShieldsPanelDataHandler::OnTabStripModelChanged(
     const TabStripModelChange& change,
     const TabStripSelectionChange& selection) {
   if (selection.active_tab_changed()) {
```

Comment:
should we check for `tab_strip_model.empty()` before this?

Reply:
I think empty() checking doesn't have any meaning.